warehouse/models/mart/gtfs/fct_daily_scheduled_shapes.sql

```diff
@@ -1,4 +1,11 @@
-{{ config(materialized='table') }}
+{{ config(
+    materialized='table',
```

Comment:
Can I ask why we're partitioning this without making it incremental? Is this for the performance of something downstream?

Reply:
This was one of several automatically surfaced BQ recommendations responsive to query patterns, rather than anything meant to address built efficiency.